### PR TITLE
WebXR - Adjusting the API to a newer version of the draft

### PR DESCRIPTION
**Pull request by RaananW**
Changing an existing definition:

- A URL to documentation or source code which provides context for the suggested changes: https://immersive-web.github.io/webxr/#idl-index

Referenced here - https://github.com/immersive-web/webxr/issues/953

cc @Manishearth

### CODE REVIEW

**Comment by RaananW** [comment]
Version was updated. the npm package WebXR has little to do with the WebXR draft and should therefore be ignored. Since WebXR is still in draft, I used 0.1 as the current version.

**Comment by RaananW** [comment]
To the future reviewer - I have followed all of the rules regarding version and naming. It doesn't fail locally, but fails in the CI.

WebXR is no npm package. it is a browser feature. So - it is not an npm package, and it is not in version 0.0.0.

**Comment by robrohan** [comment]
Hi @RaananW,

On first glance this looks good. I'll take a deeper look tonight after work.

Thank you for the PR

**Review by robrohan — APPROVED** [review]
LGTM

File: types/webxr/tslint.json [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "dtslint/dt.json"
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "npm-naming": false
+  }
```
Comment (orta) [inline types/webxr/tslint.json:5]: 👍🏻

**Comment by orta** [comment]
Cool, yeah, this feels pretty good - thanks  @RaananW

**Pull request merged**